swap/src/bitcoin/wallet.rs

```diff
@@ -22,7 +22,12 @@ use std::time::{Duration, Instant};
 use tokio::sync::{watch, Mutex};
 
 const SLED_TREE_NAME: &str = "default_tree";
-const MAX_TX_FEE: u64 = 100_000;
+
+// TODO make these values configurable
+/// Assuming we add a spread of 3% we don't want to pay more than 3% of the
+/// amount for tx fees.
+const MAX_RELATIVE_TX_FEE: f64 = 0.03;
+const MAX_ABSOLUTE_TX_FEE: u64 = 100_000;
```

Comment:
To keep the configuration small and reduce the risk of errors I propose to keep this hardcoded for the time being. 
It's too easy to get wrong and can lead to losing a lot of money.